app/assets/stylesheets/govuk-component/_breadcrumbs.scss

```diff
@@ -9,4 +9,9 @@
   }
 
   @include breadcrumbs;
+
+  a[href^="#"] {
```

Comment:
Indeed possible. We'd have to:
- Update the breadcrumb component to accept an arbitrary class (or a flag)
- Update navigation helpers to feed this class/flag in
- Add this class to static
- Possibly undo all the above if we wind up styling "normal" (ie non-taxon) breadcrumbs too (since we could do this in a more universal way in that case)

Is it worth adding this machinery? Or is it over-engineering?